lib/northbound.c

```diff
@@ -2575,10 +2575,6 @@ const char *nb_client_name(enum nb_client client)
 
 static void nb_load_callbacks(const struct frr_yang_module_info *module)
 {
-
-	if (module->ignore_cbs)
-		return;
-
```

Comment:
Because it prevents registering CLI-related callbacks. The check is only removed in this `nb_load_callbacks` function. We still don't validate the callbacks and don't call them from mgmtd.

Reply:
Why is ignore_cbs being set to true and causing cli show commands to be ignored?

Reply:
Yes, `ignore_cfg_cbs` would be a better name. I'll push an additional commit to change it.